docs/ww-manual/deep-linking.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: wasteworks
+title: Deep linking from your site
+order: 5
+user-guide: true
+---
+
+# WasteWorks deep linking
+
+Residents will most likely be finding your WasteWorks service from your own website. That may be a generic waste services information page, or another landing page for search terms such as “report missed bin” or “green garden waste”.
+
+### Important notice:
+There may be small differences in how WasteWorks functions for different councils, depending on the integrations you have selected, and the individual processes reflected in your workflow. This manual provides a **general overview** of how WasteWorks works, but may not directly reflect your unique set up.
+
+## Generic waste page
+
+On a generic page, you can link directly to the WasteWorks front page with a normal link, or embed a WasteWorks search form into the page, taking the user straight to a “Your bin days” page.
+
+#### Direct Link 
+
+Your website will have its own way of adding links.
+
+`<a href="https://your.wasteworkst.example/">Check your bin days</a>`
```

Comment:
Is the 't' in the domain names on this page to mean something?

Reply:
WorstWaste... no, just a typo. Will correct it